README.md

```diff
@@ -1,2 +1,57 @@
-# COVID19
-Using React &amp; Redux to create a mobile first application that contains comprehensive stats about Covid-19.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Covid-19 Live Statistics
+React & Redux capstone project. This project is done on module 3, week 5 of the Microverse full-stack development program.
+
+
+# Requirements for this project
+
+Please find the requirements for this project [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md).
+
+# screenshot 
+![Screenshot1](public/images/demo.jpg)
+
+
+## Technologies
+
+- HTML
+- CSS & SASS
+- JS (ES6)
+- React
+- Redux
+- JEST & React testing library
+
+## Live Demo
+You can find the live demo on [Netlify](https://effortless-travesseiro-5acbee.netlify.app/). 
+
+
```

Comment:
- [ ] Let's kindly add a section on how to get a copy of the project locally and run it as it was in the template

Reply:
Done